### PR TITLE
Update TARGETS file template

### PR DESCRIPTION
**Pull request by yiwu-arbug**
Summary:
Update template of TARGETS file according to recent changes in #4371 , #4363 and https://github.com/facebook/rocksdb/commit/dbf44c314b4adf3276afc1ca797b88944ca3162c.

Test Plan:
Regenerate TARGETS with `python buckfier/buckify_rocksdb.py`

### CODE REVIEW

**Comment by yiwu-arbug** [comment]
@abhimadan thanks for confirm!

File: buckifier/buckify_rocksdb.py [hunk 1]
```diff
@@ -109,7 +109,8 @@ def generate_targets(repo_path):
         "rocksdb_test_lib",
         src_mk.get("MOCK_LIB_SOURCES", []) +
         src_mk.get("TEST_LIB_SOURCES", []) +
-        src_mk.get("EXP_LIB_SOURCES", []),
+        src_mk.get("EXP_LIB_SOURCES", []) +
+        src_mk.get("ANALYZER_LIB_SOURCES", []),
```
Comment (anand1976) [inline buckifier/buckify_rocksdb.py]: Should this be in rocksdb_tools_lib, like in Makefile - https://github.com/facebook/rocksdb/blob/master/Makefile#L1048?

Comment (yiwu-arbug) [reply buckifier/buckify_rocksdb.py]: done

Comment (yiwu-arbug) [reply buckifier/buckify_rocksdb.py]: Looks like the analyzer lib is required by trace_analyzer_test. I'll add it back to rocksdb_test_lib

File: TARGETS [hunk 2]
```diff
@@ -919,11 +920,6 @@ ROCKS_TESTS = [
         "db/range_del_aggregator_test.cc",
         "serial",
     ],
-    [
-        "range_del_aggregator_bench",
-        "db/range_del_aggregator_bench.cc",
```
Comment (ajkr) [inline TARGETS]: can we move it to rocksdb_tools_lib?

Comment (ajkr) [reply TARGETS]: oh that's for things without a `main`, I guess. Never mind.

**Pull request closed without merging**